source/miniFastLED.h

```diff
@@ -34,6 +34,45 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "board.h"
 #include "hal.h"
 #include "light_utils.h"
+#include "settings.h"
+
+/*
+    HSV Constants
+*/
+#define HUE_RED 0
```

Comment:
I don't use them currently but it should help understand hsv2rgb better (ranges are different than described on wikis), and I imagine using those in profiles might be handy.

Reply:
This works though and gives an Okay yellow with correct dimming:
```
void yellow(led_t *currentKeyLedColors) {
  setAllKeysColorHSV(currentKeyLedColors, HUE_YELLOW, 255, 255);
} 
```